lib/components/gene/__tests__/__snapshots__/about-test.js.snap

```diff
@@ -0,0 +1,197 @@
+exports[`test renders correctly 1`] = `
+<View>
+  <View>
+    <View
+      style={
+        Object {
+          "marginLeft": 0,
+          "marginRight": 0,
+        }
+      }>
+      <Text
+        accessible={true}
+        allowFontScaling={true}
+        ellipsizeMode="tail"
+        numberOfLines={0}
+        style={
+          Array [
+            Object {
+              "fontSize": 17,
+            },
+            Object {
+              "fontSize": 16,
+              "lineHeight": 20,
+              "marginBottom": 20,
+            },
+            Object {
+              "fontFamily": "AGaramondPro-Regular",
+            },
+          ]
+        }>
+        This is some example text
+      </Text>
+    </View>
+    <View
+      style={
+        Array [
+          Object {
+            "backgroundColor": "#e5e5e5",
+            "height": 1,
+            "marginLeft": -20,
+            "marginRight": -20,
+          },
+          Object {
+            "marginBottom": 20,
+          },
+        ]
+      } />
+  </View>
+</View>
+`;
+
+exports[`test shows trending artists correctly 1`] = `
+<View>
+  <View>
+    <View
+      style={
+        Object {
+          "marginLeft": 0,
+          "marginRight": 0,
+        }
+      }>
+      <Text
+        accessible={true}
+        allowFontScaling={true}
+        ellipsizeMode="tail"
+        numberOfLines={0}
+        style={
+          Array [
+            Object {
+              "fontSize": 17,
+            },
+            Object {
+              "fontSize": 16,
+              "lineHeight": 20,
+              "marginBottom": 20,
+            },
+            Object {
+              "fontFamily": "AGaramondPro-Regular",
+            },
+          ]
+        }>
+        This is some example text
+      </Text>
+    </View>
+    <View
+      style={
+        Array [
+          Object {
+            "backgroundColor": "#e5e5e5",
+            "height": 1,
+            "marginLeft": -20,
+            "marginRight": -20,
+          },
+          Object {
+            "marginBottom": 20,
+          },
+        ]
+      } />
+  </View>
+  <View
```

Comment:
same as above but now we have related artists

![screen shot 2016-11-28 at 16 54 57](https://cloud.githubusercontent.com/assets/49038/20677609/77016472-b58b-11e6-97b3-ba587ab6813f.png)